x-pack/plugins/cloud_security_posture/server/routes/configuration/update_rules_configuration.test.ts

```diff
@@ -169,21 +170,13 @@ describe('Update rules configuration API', () => {
       ],
     } as unknown as SavedObjectsFindResponse<CspRuleSchema>;
     const cspConfig = await createRulesConfig(cspRules);
-    expect(cspConfig).toMatchObject({ activated_rules: { cis_k8s: [] } });
-  });
-
-  it('validate converting rules config object to Yaml', async () => {
-    const cspRuleConfig = { activated_rules: { cis_k8s: ['1.1.1', '1.1.2'] } };
-
-    const dataYaml = convertRulesConfigToYaml(cspRuleConfig);
-
-    expect(dataYaml).toEqual('activated_rules:\n  cis_k8s:\n    - 1.1.1\n    - 1.1.2\n');
+    expect(cspConfig).toMatchObject({ data_yaml: { activated_rules: { cis_k8s: [] } } });
   });
 
   it('validate adding new data.yaml to package policy instance', async () => {
     const packagePolicy = createPackagePolicyMock();
 
-    const dataYaml = 'activated_rules:\n  cis_k8s:\n    - 1.1.1\n    - 1.1.2\n';
+    const dataYaml = 'data_yaml:\n  activated_rules:\n  cis_k8s:\n    - 1.1.1\n    - 1.1.2\n';
```

Comment:
indentation looks off to me
can you verify?

Reply:
seems that `activated_rules` and `cis_k8s` have the same indentation

Reply:
++
```suggestion
    const dataYaml = 'data_yaml:\n  activated_rules:\n    cis_k8s:\n      - 1.1.1\n      - 1.1.2\n';
```